.vale/styles/RedHat/Headings.yml

```diff
@@ -105,11 +108,14 @@ exceptions:
   - Makefile
   - Mattermost
   - Maven
+  - MicroProfile
   - Microsoft
   - Middleware
   - Minikube
   - Minishift
   - Mirantis
+  - MongoDB
+  - Mutual TLS
```

Comment:
Also known as `mTLS`. Would it make sense to add that, too?

Reply:
Done. Added mTLS to the definitions.yml file. :+1: